hands-on/gemv/1-gemv-omp/Makefile

```diff
@@ -1,9 +1,15 @@
 include ../../make.inc
 
 name=gemv-omp
+
+all: ${name}.x ${name}.f
```

Comment:
Let's name the binary for fortran as ${name}-f.x

Reply:
name of the fortran binary has been updated

Reply:
In the above line the one listed after "all" is wrong

Reply:
${name}.f has been correct to ${name}-f.x